### PR TITLE
doc/boards: fix minor editorial mistakes

### PR DESCRIPTION
**Pull request by krzysztof-cabaj**
### Contribution description

This PR fix some minor editorial mistakes:
- nucleo-f767 - add ending bracket in ADCs details,
- nucleo-u575zi-q - fix wrong board name - change nucleo-l552ze-q to u575zi-q,
- stmf469i-disco - change inch sign (`"`) which corrupt bottom of the doc page to (`''`).

### Testing procedure

```
make doc
xdg-open doc/doxygen/html/group__boards__nucleo-f767zi.html 
xdg-open doc/doxygen/html/group__boards__nucleo-u575zi-q.html 
xdg-open doc/doxygen/html/group__boards__nucleo-stm32f469i-disco.html 
```

### Issues/PRs references

None

### CODE REVIEW

File: boards/nucleo-u575zi-q/doc.txt [hunk 1]
```diff
@@ -37,7 +33,7 @@ of Flash.
 
 ### Flashing the Board Using OpenOCD
 
-The ST Nucleo-L552ZE-Q board includes an on-board ST-LINK programmer and can be
+The ST Nucleo-L575ZI-Q board includes an on-board ST-LINK programmer and can be
```
Comment (crasbe) [inline boards/nucleo-u575zi-q/doc.txt:36]:
```suggestion
The ST Nucleo-U575ZI-Q board includes an on-board ST-LINK programmer and can be
```

You missed the L here :)

Comment (krzysztof-cabaj) [reply boards/nucleo-u575zi-q/doc.txt]: Thanks for spotting this.
Fixed!

File: boards/stm32f469i-disco/doc.txt [hunk 2]
```diff
@@ -81,7 +81,7 @@ Also provides some system signals and power.
 - 1 User button.
 - 4 LEDs.
 - 3 Digital microphones
-- 1 LCD Color 4"
+- 1 LCD Color 4''
```
[the remark this answers is not in the document]
Comment (dylad) [reply boards/stm32f469i-disco/doc.txt:84]: Seems like the " symbol is causing some corruption.
Maybe replaces it by `in` or `inches` directly instead.

[the remark this answers is not in the document]
Comment (crasbe) [reply boards/stm32f469i-disco/doc.txt:84]: Or you could use the HTML escape character &quot;
I tested that and it produces the correct output in the HTML file.

[the remark this answers is not in the document]
Comment (benpicco) [reply boards/stm32f469i-disco/doc.txt:84]: Does `″` work?

[the remark this answers is not in the document]
Comment (dylad) [reply boards/stm32f469i-disco/doc.txt:84]: @benpicco 
Copie d'écran_20240507_183108

That's in current master with `"` already

[the remark this answers is not in the document]
Comment (dylad) [reply boards/stm32f469i-disco/doc.txt:84]: I still think we should go with either `in` or `inches`

[the remark this answers is not in the document]
Comment (benpicco) [reply boards/stm32f469i-disco/doc.txt:84]: I mean `″` not `"`

[the remark this answers is not in the document]
Comment (krzysztof-cabaj) [reply boards/stm32f469i-disco/doc.txt]: I try to use the first mentioned char ... but cannot include it in doc file. Direct copy from Windows gives strange effects. What is interesting at my Debian 12, in Firefox both signs are identical. I change this to text `inches` as @dylad  suggested.

File: boards/nucleo-u575zi-q/doc.txt [hunk 3]
```diff
@@ -5,14 +5,10 @@
...
 ## Overview
 
-The Nucleo-L552ZE-Q is a board from ST's Nucleo family supporting the ARM Cortex-M33
+The Nucleo-U575ZI-Q is a board from ST's Nucleo family supporting the ARM Cortex-M33
 STM32U575ZIT6Q ultra-low-power microcontroller with TrustZone, 768KiB of RAM and 2MiB
 of Flash.
 
-## Hardware
-
-![Nucleo144 L552ZE-Q](https://www.st.com/bin/ecommerce/api/image.PF271812.en.feature-description-include-personalized-no-cpn-medium.jpg)
```
Comment (benpicco) [inline boards/nucleo-u575zi-q/doc.txt:14]: Is this not the right picture?

Comment (crasbe) [reply boards/nucleo-u575zi-q/doc.txt:14]: Is is the right picture, but it was the wrong caption. The picture could probably stay and the caption be corrected to "Nucleo-144 U575ZI-Q".

**Comment by krzysztof-cabaj** [comment]
I actually cut out too much :( ... I compare this image with the one in the Board Manual and they seams to be identical.
I change picture description.

**Review by benpicco — APPROVED** [review]

**Comment by krzysztof-cabaj** [comment]
@benpicco, @crasbe, @dylad Thanks for valuable comments and PR review!

**Pull request merged**